source/introduction.rst

```diff
@@ -0,0 +1,90 @@
+
+Perun’s state-channel virtualization
+````````````````````````````````````
+To connect users that do not have a joint open state-channel, existing state-channels can be composed to form so-called virtual channels. These virtual channels are created and closed off-chain over the state-channel network intermediaries. Once opened, the virtual channel is updated directly off-chain between the two connected end users.
+
+.. image:: ./images/introduction/perun_protocol_overview.svg
+  :align: Center
+  :alt: Image not available
+
+Blockchain-agnostic
+```````````````````
+Its modular design enables the flexible integration of Perun’s state-channel technology into any Blockchain or traditional ledger system. 
+
+Interoperability
+````````````````
+The blockchain agonistic design and state-channel virtualization enable transaction and smart contract execution even across different blockchains (cross-chain functionality).
+
+
+High security
+`````````````
+The Perun protocol specifications have been mathematically proven using the latest methods of security research.
+
+The Perun protocol can be used for a wide range of applications in different areas such as finance/FinTech, mobility, energy, e-commerce, telecommunication and any other use case where direct microtransactions are needed.
+
+
+The Hyperledger Labs Project
+----------------------------
+
+As a first step, we are developing a secure and efficient standalone payment application within the Perun Hyperledger Labs project. The labs project currently consists of the following main parts that together form the Perun Framework.
+
+.. image:: ./images/introduction/perun_framework.svg
+  :align: Center
+  :alt: Image not available
+
+perun-eth-contracts
+```````````````````
+This provides the Ethereum smart contracts required for implementing the Perun protocol.
+
+Link to the project on GitHub: https://github.com/hyperledger-labs/perun-eth-contracts
+
+go-Perun
+`````````
+An SDK that implements core components of the Perun protocol (state-channel proposal protocol, the state machine that supports persistence and a watcher) and an Ethereum blockchain connector. It is designed to be blockchain agnostic.
+
+Link to the project on GitHub: https://github.com/hyperledger-labs/go-perun
+
+Perun-node
+``````````
+Perun-node is a multiuser node which provides a common interface for using state channels. Perun-node generally aims to execute perun protocol by implementing key management, user API, peer ID provider and user session on top of state channel client implemented by go-perun. You can use perun-node to open, transact, settle and close state channels.
+
+Link to the project on GitHub: https://github.com/hyperledger-labs/perun-node
+
+Current functionalities available
```

Comment:
Also already available in go-perun:
- Channel persistence and restoring.
- Sub-channels for locking funds to a specific channel application